app/assets/javascripts/select-all.js

```diff
@@ -5,12 +5,22 @@
     this.start = function (element) {
       function addEventListener() {
         var selectAll = element.find('#select_all');
+        var selectAllPages = element.find('#select_all_pages');
+        var selectAllPagesContainer = selectAllPages.closest('.if-js-hide');
```

Comment:
`var selectAllPagesContainer = element.find('.checkbox.if-js-hide');` should just work actually so I might switch it to that.